modules/tftp/tftp_api.rb

```diff
@@ -34,6 +34,11 @@ def create_default(variant)
       end
     end
 
+    post "/fetch_boot_image" do
+      log_halt(400, "TFTP: Wrong input parameters given. ") unless [params[:path], params[:url], params[:files]].all?
```

Comment:
Is the empty space intended?

Nit: I'd leave an empty line after it since essentially this is a return statement
```suggestion
      log_halt(400, "TFTP: Wrong input parameters given.") unless [params[:path], params[:url], params[:files]].all?

```

Reply:
Done.